01_install_requirements.sh

```diff
@@ -34,31 +34,50 @@ sudo dnf -y upgrade
 # shellcheck disable=SC1091
 source /etc/os-release
 export DISTRO="${ID}${VERSION_ID%.*}"
-if [[ $DISTRO == "centos8" ]]; then
-    echo "CentOS is not supported anymore. Please switch to CentOS Stream / RHEL / Rocky Linux"
-    sudo dnf -y install epel-release dnf --enablerepo=extras
-elif [[ $DISTRO == "rhel8" ]]; then
-    # Enable EPEL for python3-passlib and python3-bcrypt required by metal3-dev-env
-    sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
-
-    if sudo subscription-manager repos --list-enabled 2>&1 | grep "ansible-2-for-rhel-8-x86_64-rpms"; then
-      # The packaged 2.x ansible is too old for compatibility with metal3-dev-env
-      sudo dnf erase -y ansible
-      sudo subscription-manager repos --disable=ansible-2-for-rhel-8-x86_64-rpms
-    fi
-fi
```

Comment:
RHEL8 still works fine for me and RHEL9 isn't yet released, so I wonder if we can maintain RHEL8 for a bit longer?